sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/tool/proxy/IntegrationAdapterProxy.java

```diff
@@ -126,13 +126,13 @@ public void initializeByType(String integrationTypeName, List<Integration> integ
                                                             .collect(Collectors.toList());
 
         // make sure all of those have mandatory settings in place
-        long validIntegrationsCount = enabledIntegrations.stream()
+        long invalidIntegrationsCount = enabledIntegrations.stream()
```

Comment:
Why would you change variable name?

Reply:
`cause a game was changed, for now we are checking that values are not valid

Reply:
Still dont get it. Judging by comments and other method names this is something i still can't understand.